### PR TITLE
add side bar content and table for prevalence page

### PR DESCRIPTION
**Pull request by mahtabiltarabian**

### CODE REVIEW

**Review by dptolle — CHANGES_REQUESTED** [review]

File: src/app/prevalence/components/PrevalenceMainContent.tsx [hunk 1]
```diff
@@ -1,28 +1,103 @@
...
+            style={{
+                padding: theme.spacing(3),
+                display: "flex",
+                justifyContent: "center",
+                width: "100%",
             }}
         >
-            {heading}
-        </Typography>
+            <div style={{ maxWidth: "70%" }}>
```
[the remark this answers is not in the document]
Comment (mahtabiltarabian) [reply src/app/prevalence/components/PrevalenceMainContent.tsx]: yes, I used box instead div

File: src/locales/de/PrevalencePage.json [hunk 2]
```diff
@@ -1,4 +1,45 @@
 {
   "PREVALENCE": "Prävalenz",
-  "FILTER_SETTINGS": "Filter Einstellungen"
+  "FILTER_SETTINGS": "Filter Einstellungen",
+  "FILTER": "Filter",
+  "SELECT_ALL": "Selecteer Alles",
+  "DESELECT_ALL": "Deselecteer Alles",
```
Comment (dptolle) [inline src/locales/de/PrevalencePage.json:6]: Deselect: Alle abwählen

File: src/locales/de/PrevalencePage.json [hunk 3]
```diff
@@ -1,4 +1,45 @@
 {
   "PREVALENCE": "Prävalenz",
-  "FILTER_SETTINGS": "Filter Einstellungen"
+  "FILTER_SETTINGS": "Filter Einstellungen",
+  "FILTER": "Filter",
+  "SELECT_ALL": "Selecteer Alles",
```
Comment (dptolle) [inline src/locales/de/PrevalencePage.json:5]: Select all: alle auswählen

File: src/app/prevalence/components/SelectionContext.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    setSelectedAnimalSpecies: (species: string[]) => void;
+}
+
+// Create the context with an initial undefined value
+const SelectionContext = createContext<SelectionContextData | undefined>(
+    undefined
+);
+
+// Create a custom hook to use the context
+export const useSelection = (): SelectionContextData => {
```
Comment (dptolle) [inline src/app/prevalence/components/SelectionContext.tsx:17]: PLease rename the hook to: usePrevalenceFilters, or something more specific - useSelection is too general

File: src/app/prevalence/components/PrevalenceSideContent.tsx [hunk 5]
```diff
@@ -1,5 +1,233 @@
...
+    "WIEDERKAEUER",
+    "WILDGEFLUEGEL",
+    "WILDSCHWEIN",
+    "WILDWIEDERKAEUER",
+];
 
 export function PrevalenceSideContent(): JSX.Element {
-    return <></>;
+    // This already has a return type specified.
+    const {
```
Comment (dptolle) [inline src/app/prevalence/components/PrevalenceSideContent.tsx:76]: Please encapsulate all the functionality into a hook.  The PrevalenceSideContent component should then use the hook to get model values and actions (functions) which are then inserted into the jsx.

**Review by dptolle — CHANGES_REQUESTED** [review]
Looks good.  Just some minor changes needed

**Review by dptolle — APPROVED** [review]

**Pull request merged**